Build on OS X fails because of libpq

Error message (after updating submodules and install pq):
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/Users/markus/Projekte/Opossum/Git/third_party/PQ_INCLUDE_DIR
   used as include directory in directory /Users/markus/Projekte/Opossum/Git/third_party
```

Full log:
```
[:~/Projekte/Opossum/Git] master(+1/-0)+ ± gl | head -n1
commit 89a0df9da9b613a185145569727f4a5f9d21f713
[:~/Projekte/Opossum/Git] master(+1/-0)+ ± git submodule update --init --recursive
[:~/Projekte/Opossum/Git] master(+1/-0)+ ± brew install libpq
Warning: libpq 10.3 is already installed and up-to-date
To reinstall 10.3, run `brew reinstall libpq`
[:~/Projekte/Opossum/Git] master(+1/-0)+ ± mkdir build
[:~/Projekte/Opossum/Git] master(+1/-0)+ ± cd !$
cd build
/Users/markus/Projekte/Opossum/Git/build
[:~/Projekte/Opossum/Git/build] master(+1/-0)+ ± cmake -DCMAKE_C_COMPILER=/usr/local/opt/llvm/bin/clang-5.0 -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm/bin/clang++ ..
-- The C compiler identification is Clang 5.0.1
-- The CXX compiler identification is Clang 5.0.1
-- Check for working C compiler: /usr/local/opt/llvm/bin/clang-5.0
-- Check for working C compiler: /usr/local/opt/llvm/bin/clang-5.0 -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/local/opt/llvm/bin/clang++
-- Check for working CXX compiler: /usr/local/opt/llvm/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found C++ stdlib filesystem library: lib=/usr/local/Cellar/llvm/5.0.1/lib/libc++experimental.a
-- WARNING: Numa library not found.
-- Try: 'sudo yum install numactl numactl-devel' (or sudo apt-get install libnuma libnuma-dev)
-- Found tbb library: inc=/usr/local/include, lib=/usr/local/lib/libtbb.dylib
-- Found readline library: inc=/usr/local/opt/readline/include, lib=/usr/local/opt/readline/lib/libreadline.dylib
-- Looking for wsyncup in /usr/lib/libcurses.dylib
-- Looking for wsyncup in /usr/lib/libcurses.dylib - found
-- Looking for cbreak in /usr/lib/libncurses.dylib
-- Looking for cbreak in /usr/lib/libncurses.dylib - found
-- Found Curses: /usr/lib/libncurses.dylib
-- Found sqlite3 library: inc=/usr/include, lib=/usr/lib/libsqlite3.dylib
-- WARING: pq library not found.
-- Try: 'sudo yum install libpq-dev' (or sudo apt-get install libpq-dev)
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   system
--   thread
--   chrono
--   date_time
--   atomic
-- Found llvm library: inc=/usr/local/opt/llvm/include, lib=/usr/local/opt/llvm/lib/libLLVM.dylib
-- Found PythonInterp: /usr/bin/python (found version "2.7.10")
-- Building with JIT support
-- Building without NUMA support
-- The ASM compiler identification is Clang
-- Found assembler: /usr/local/opt/llvm/bin/clang-5.0
-- Adding operators/jit_operator/jit_operations.cpp to LLVM bundle
-- Adding operators/jit_operator/jit_operations.hpp to LLVM bundle
-- Adding operators/jit_operator/jit_types.cpp to LLVM bundle
-- Adding operators/jit_operator/jit_types.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/abstract_jittable.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/abstract_jittable_sink.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_compute.cpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_compute.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_expression.cpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_expression.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_filter.cpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_filter.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_read_tuple.cpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_read_tuple.hpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_write_tuple.cpp to LLVM bundle
-- Adding operators/jit_operator/operators/jit_write_tuple.hpp to LLVM bundle
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/Users/markus/Projekte/Opossum/Git/third_party/PQ_INCLUDE_DIR
   used as include directory in directory /Users/markus/Projekte/Opossum/Git/third_party

-- Configuring incomplete, errors occurred!
See also "/Users/markus/Projekte/Opossum/Git/build/CMakeFiles/CMakeOutput.log".
[:~/Projekte/Opossum/Git/build] master(+1/-0)+ ±
```

--- a/cmake/FindPQ.cmake
+++ b/cmake/FindPQ.cmake
@@ -5,11 +5,11 @@
 #   PQ_LIBRARY     : library path of postgres' pq
 
 FIND_PATH(PQ_INCLUDE_DIR NAME libpq-fe.h HINTS
-    $ENV{HOME}/local/include /opt/local/include /usr/local/include /usr/include /usr/local/include/postgresql /usr/include/postgresql
+    $ENV{HOME}/local/include /opt/local/include /usr/local/include /usr/include /usr/local/include/postgresql /usr/include/postgresql /usr/local/opt/libpq/include
 )
 
 FIND_LIBRARY(PQ_LIBRARY NAME pq HINTS
-    "$ENV{LIB_DIR}/lib" "$ENV{LIB}/lib" "$ENV{HOME}/local/lib64" "$ENV{HOME}/local/lib" /usr/local/lib64 /usr/local/lib /opt/local/lib64 /opt/local/lib /usr/lib64 /usr/lib
+    "$ENV{LIB_DIR}/lib" "$ENV{LIB}/lib" "$ENV{HOME}/local/lib64" "$ENV{HOME}/local/lib" /usr/local/lib64 /usr/local/lib /opt/local/lib64 /opt/local/lib /usr/lib64 /usr/lib /usr/local/opt/libpq/lib
 )
 
 IF(PQ_INCLUDE_DIR AND PQ_LIBRARY)

